<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>773</width>
      <height>276</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="6">
      <widget class="QDoubleSpinBox" name="VX5">
       <property name="minimum">
        <double>-99999999999999995280522225138166806691251291352861698530421623488512.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000004384584304507619735463404765184.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSpinBox" name="numeroc"/>
     </item>
     <item row="6" column="5">
      <widget class="QDoubleSpinBox" name="Y4">
       <property name="minimum">
        <double>-99999999999999999322094867436162797646170844194406400.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000000620008645040778319495168.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="cuerpo2">
       <property name="text">
        <string>Cuerpo 2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="cuerpo4">
       <property name="text">
        <string>Cuerpo 4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="y">
       <property name="text">
        <string>Posicion Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="y2">
       <property name="text">
        <string>Posicion Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="masa2">
       <property name="text">
        <string>Masa</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="vy">
       <property name="text">
        <string>Velocidad Yo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLabel" name="x">
       <property name="text">
        <string>Posicion X</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QDoubleSpinBox" name="Y5">
       <property name="minimum">
        <double>-99999999999999996863366107917975552.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999999999538762658202121142272.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="x2">
       <property name="text">
        <string>Posicion X</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QDoubleSpinBox" name="VX4">
       <property name="minimum">
        <double>-100000000000000007629769841091887003294964970946560.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000000102350670204085511496304388135324745728.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QDoubleSpinBox" name="Vyo">
       <property name="minimum">
        <double>-10000000000000000470601344959054695891559601407866630764278709534898249531392.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999999999926539781176481198923508803215199467887262646419780362305536.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Fin</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QDoubleSpinBox" name="Vxo">
       <property name="minimum">
        <double>-10000000000000000139372116959414099130712064.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000000102350670204085511496304388135324745728.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QDoubleSpinBox" name="VX3">
       <property name="minimum">
        <double>-9999999999999999493871352970740188669636450110134100730839040.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000000213204190094543968723012578712679649467743338496.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QDoubleSpinBox" name="Yo">
       <property name="minimum">
        <double>-99999999999999999322094867436162797646170844194406400.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000000146306952306748730309700429878646550592786107871697963642511482159104.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QDoubleSpinBox" name="VY5">
       <property name="minimum">
        <double>-9999999999999999931398190359470212947659194368.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000048346692115553659057528394845890514255872.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="Xo">
       <property name="minimum">
        <double>-9999999999999999464902769475481793196872414789632.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999999999438119489974413630815797154428513196965888.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="X4">
       <property name="minimum">
        <double>-1000000000000000078291540404596243842305360299886116864.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999999999322094867436162797646170844194406400.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QDoubleSpinBox" name="VY4">
       <property name="minimum">
        <double>-9999999999999999464902769475481793196872414789632.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999999999932209486743616279764617084419440640.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QDoubleSpinBox" name="YI">
       <property name="minimum">
        <double>-1000000000000000078291540404596243842305360299886116864.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000000213204190094543968723012578712679649467743338496.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QDoubleSpinBox" name="VY3">
       <property name="minimum">
        <double>-1000000000000000044885712678075916785549312.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999999999932209486743616279764617084419440640.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="XI">
       <property name="minimum">
        <double>-10000000000000000139372116959414099130712064.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999999999322094867436162797646170844194406400.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="X3">
       <property name="minimum">
        <double>-99999999999999999209038626283633850822756121694230455365568299008.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000044885712678075916785549312.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="X5">
       <property name="minimum">
        <double>-100000000000000007629769841091887003294964970946560.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999999997748809823456034029568.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="cuerpo3">
       <property name="text">
        <string>Cuerpo 3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QDoubleSpinBox" name="masa">
       <property name="maximum">
        <double>9999999999999999827367757839185598317239782875580932278577147150336.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="cuerpo5">
       <property name="text">
        <string>Cuerpo 5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="numc">
       <property name="text">
        <string>#Cuerpos</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QDoubleSpinBox" name="Y3">
       <property name="minimum">
        <double>-999999999999999993220948674361627976461708441944064.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000048346692115553659057528394845890514255872.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="vx">
       <property name="text">
        <string>Velocidad Xo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cuerpo1">
       <property name="text">
        <string>Cuerpo 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Bola">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>480</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagen.qrc">:/bola1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bola1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/bola.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bola_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="bola3.qrc">:/bola3.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bola_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="bola2.qrc">:/bola2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bola_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="bola4.qrc">:/bola4.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="bola2.qrc"/>
  <include location="imagen.qrc"/>
  <include location="bola4.qrc"/>
  <include location="bola3.qrc"/>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
